DRAFT_Connectivity.Rmd

```diff
@@ -1,266 +1,280 @@
+
+Framstad, E., Eide, N.E., Eide, W., Klanderud, K., Kolstad, A., Töpper, J. & Vandvik, V. 2022. Vurdering av økologisk tilstand for fjell i Norge i 2021. NINA Rapport 2050. Norsk institutt for naturforskning.
+
+
+## Analyses {#analyses-con}
+### Data sets {#datasets-con}
+Two main data sets were included. 
+
+A. The infrastructure index (Erikstad et al. 2023) and; 
+
+B. the wetland model (Bakkestuen et al. 2023, Bakkestuen et al. _In prep._) 
+
+
+#### Data set A - Infrastructure index {#data-a-con}
+The infrastructure index utilized in this analysis was derived from data sources available through Google Earth Engine (GEE). The index quantifies the presence and distribution of key infrastructure elements such as roads, buildings, and related human-made structures across Norway. The data were available for the years 2003, 2013, and 2023, facilitating an assessment of how infrastructure has evolved in terms of both magnitude and distribution over time. It is important to note that this data forms a fundamental component in the calculation of the connectivity indicator.
+
+#### Data set B - Wetland model {#data-b-con}
+The wetland model used in this study was developed based on the integration of Sentinel-2 satellite imagery and LiDAR data, utilizing deep learning techniques. The model provides high-resolution information regarding the distribution and extent of wetland areas within southern Norway, specifically for the year 2020. By fusing spectral and elevation data, this model offers significant improvements over conventional, manually-digitized land cover maps. The wetland model is a critical component of the connectivity analysis, as it aids in the identification of ecologically relevant areas. The data set B for South-Norway is already accessible [here](https://kartkatalog.geonorge.no/metadata/aapen-vtmark-i-soer-norge-basert-p-natur-i-norge-nin-typologi/6b5a34f9-c23d-4cca-a51b-01251c217cb1).
+
+### Scaled indicator values {#scaled-values-con}
+The connectivity indicator was created by examining the interplay between infrastructure and wetland areas. We initially calculated the average distance between wetland patches using the wetland model, representing a baseline measure of natural connectivity in the absence of infrastructure (i.e., the reference condition). Subsequently, the infrastructure index was introduced, portraying the extent of human-made structural elements within the landscape.
+The scaled indicator is the quotient of the values. 
+
+
+### Uncertainty {#uncertainty-con}
+The current state of the indicator is as a geospatial data set with one scaled indicator value per mire polygon, and no errors around this value. Future improvement must include aggregating these indicator values to regions. The uncertainties around these aggregated indicator values could then be represented by the spatial variation in the indicator values.  
```

Comment:
This subchapter should explain how errors are estimated. No such errors ar estimated yet for this indicator, but I have a suggestion for how: #144